doc/build.md

```diff
@@ -120,6 +120,28 @@ cargo build --release
 **Want to run or hack on Mayastor?** *You need more configuration!* See
 [running][doc-running], then [testing][doc-testing].
 
+### Customizing the rust toolchain
```

Comment:
I feel like this section could be reemphasized to a simple quote around line 98?


```markdown
> If you are unable to use the Nix provided Rust for some reason, there are `norust` and `nospdk` arguments to Nix shell. `nix-shell --arg norust true`
```